<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsynOneBufferedAI_TDtrClass</class>
 <widget class="QDialog" name="AsynOneBufferedAI_TDtrClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>515</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>762</width>
    <height>515</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>762</width>
    <height>515</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Asynchronous One Buffered AI with Trigger Delay to Start and Delay toStop</string>
  </property>
  <widget class="QPushButton" name="btnGetData">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>605</x>
     <y>486</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Data</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblXCoordinateEnd">
   <property name="geometry">
    <rect>
     <x>623</x>
     <y>410</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>10Sec</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblShift">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>483</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Shift:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblYCoordinateMid">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>193</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtDivValue">
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>480</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>430</y>
     <width>475</width>
     <height>38</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblYCoordinateMax">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>42</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10.0V</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblDiv">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>483</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Div:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblColor">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>430</y>
     <width>61</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>Color of
channels:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblYCoordinateMin">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>347</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-10.0V</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="sldShift">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>481</y>
     <width>128</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtShiftValue">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>480</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblDivUnit">
   <property name="geometry">
    <rect>
     <x>437</x>
     <y>483</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfigure">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>605</x>
     <y>447</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="graphFrame">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>41</y>
     <width>660</width>
     <height>324</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblXCoordinateStart">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>411</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0Sec</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblShiftUnit">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>484</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="sldDiv">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>458</x>
     <y>480</y>
     <width>128</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_triggerPoint">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>121</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>121</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>0 ms Triggered.</string>
   </property>
  </widget>
  <widget class="QFrame" name="triggerPointFlag">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>31</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:url(:/AsynOneBufferedAI_TDtrtp/Resources/trigger.png)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>triggerPointFlag</zorder>
  <zorder>btnGetData</zorder>
  <zorder>lblXCoordinateEnd</zorder>
  <zorder>lblShift</zorder>
  <zorder>lblYCoordinateMid</zorder>
  <zorder>edtDivValue</zorder>
  <zorder>listWidget</zorder>
  <zorder>lblYCoordinateMax</zorder>
  <zorder>lblDiv</zorder>
  <zorder>lblColor</zorder>
  <zorder>lblYCoordinateMin</zorder>
  <zorder>sldShift</zorder>
  <zorder>edtShiftValue</zorder>
  <zorder>lblDivUnit</zorder>
  <zorder>btnConfigure</zorder>
  <zorder>graphFrame</zorder>
  <zorder>lblXCoordinateStart</zorder>
  <zorder>lblShiftUnit</zorder>
  <zorder>sldDiv</zorder>
  <zorder>label_triggerPoint</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
